src/components/input-dropdown/input-dropdown.component.html

```diff
@@ -23,11 +23,11 @@
 
 <ux-menu #menu menuClass="select-menu"
          (opened)="onMenuOpen()"
-         (closed)="onMenuClose()"
-         [attr.aria-expanded]="dropdownOpen">
+         (closed)="onMenuClose()">
 
     <div [style.max-height]="_maxHeight"
-         [style.width.px]="button.offsetWidth">
+         [style.width.px]="button.offsetWidth"
+         role="group">
```

Comment:
I don't think `role="group"` is correct here. Better would be to remove `role="menu"` from the parent ux-menu, or use `role="menuitemradio"` on the radio buttons.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role#example_2_menubar_submenu_option_picker